src/SFA.DAS.AssessorService.Application.Api.UnitTests/Validators/ExternalApi/Epas/DeleteBatchEpaRequestValidator/WhenDeletingDraftCertificate.cs

```diff
@@ -0,0 +1,41 @@
+﻿using FizzWare.NBuilder;
+using FluentAssertions;
+using FluentValidation.Results;
+using NUnit.Framework;
+using SFA.DAS.AssessorService.Api.Types.Models.ExternalApi.Epas;
+using SFA.DAS.AssessorService.Domain.JsonData;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.AssessorService.Application.Api.UnitTests.Validators.ExternalApi.Epas.DeleteBatchEpaRequestValidator
+{
+    public class WhenDeletingDraftCertificate : DeleteBatchEpaRequestValidatorTestBase
```

Comment:
Looking at the bast how is one deleted and the other further down with the same base submitted?

Reply:
Can you double check as to me they have different Certificate details

Reply:
Ok I see that now - but it's really hard to tell them apart